environment/swarmball_env.py

```diff
@@ -4,17 +4,24 @@
 from .simulation.simulation import SwarmBallSimulation
 
 class SwarmBall(gym.Env):
-    def __init__(self):
-        self.sim = SwarmBallSimulation()
+    def __init__(self, acc_factor=0.25, thresh_amount=3, **kwargs):
+        self.sim = SwarmBallSimulation(**kwargs)
```

Comment:
Dlaczego kwargs? Co to robi tak w zasadzie?

Reply:
Po prostu jak chcesz przekazać jakiekolwiek argumenty do konstruktora symulacji to teraz możesz przez kwargs. Krótko mówiąc wszystkie argumenty klucz-wartość przekazane do konstruktora środowiska jeżeli nie należą do niego będą przekazane dalej do symulacji.

Reply:
Uuuu, fajne